tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -892,8 +1101,107 @@ class MklConvOp : public OpKernel {
   const int kOutputIndex_Dst = 0, kOutputIndex_Filter = 1;
   const int kDilationH = 0, kDilationW = 1;
 
+#ifdef ENABLE_MKLDNN_V1
   // Allocate persistent tensors for cached filter data and
   // cached filter memory descriptor (data format)
+  void AllocatePersistentTensor(OpKernelContext* context,
+                                const ConvFwdPd& conv_prim_desc,
+                                Tensor** filter_tensor,
+                                const MklDnnShape& filter_mkl_shape) {
```

Comment:
Please try to merge with existing original code (same or the rest of the PR). This one can call the one with three parameters then do extra processing.

Reply:
Done.